doc/source/user_guide/fields/reduction.rst

```diff
@@ -3,350 +3,302 @@
+
+  >>> reduction.area_average(
+  ...     expression="AbsolutePressure",
+  ...     locations=solver.setup.boundary_conditions.velocity_inlet,
+  ... )
+  101325.0
 
-You can calculate the area average of "Absolute Pressure" over the entire set of velocity
-inlets as shown:
+Object-Oriented:
 
 .. code-block:: python
 
   >>> solver.fields.reduction.area_average(
-  >>>   expression="AbsolutePressure",
-  >>>   locations=solver.settings.setup.boundary_conditions.velocity_inlet,
-  >>> )
-  101325.0000000001
+  ...     expression="AbsolutePressure",
+  ...     locations=solver.settings.setup.boundary_conditions.velocity_inlet,
+  ... )
+  101325.0
 
-You can calculate the area integrated average of "Absolute Pressure" over the velocity inlet 1
-as shown:
+**Example: Minimum Across Multiple Solvers**
 
 .. code-block:: python
 
-  >>> solver.fields.reduction.area_integral(
-  >>>   expression="AbsolutePressure",
-  >>>   locations=[solver.settings.setup.boundary_conditions.velocity_inlet["inlet1"]],
-  >>> )
-  0.07665669042888468
+  >>> reduction.minimum(
+  ...     expression="AbsolutePressure",
+  ...     locations=[
+  ...         solver1.setup.boundary_conditions.pressure_outlet,
+  ...         solver2.setup.boundary_conditions.pressure_outlet,
+  ...     ],
+  ... )
+  101325.0
 
-You can calculate the geometric centroid of the velocity inlet 2 as shown:
+**Example: Using Boundary Abstractions**
 
 .. code-block:: python
 
-  >>> solver.fields.reduction.centroid(
-  >>>   locations=[solver.settings.setup.boundary_conditions.velocity_inlet["inlet2"]]
-  >>> )
-  x: -0.001000006193379666
-  y: -0.002999999999999999
-  z: 0.001500047988232209
+  >>> reduction.minimum(
+  ...     expression="AbsolutePressure",
+  ...     locations=[
+  ...         VelocityInlets(settings_source=solver) for solver in [solver1, solver2]
+  ...     ],
+  ... )
+  101325.0
 
-You can calculate the moment vector about a single-valued expression
-for the specified locations as shown:
+**Example: Geometric centroid of the velocity inlet 2**
 
 .. code-block:: python
 
-  >>> solver.fields.reduction.moment(
-  >>>   expression="Force(['wall'])",
+  >>> cent = reduction.centroid(
   >>>   locations=[solver.settings.setup.boundary_conditions.velocity_inlet["inlet2"]]
   >>> )
-  [ 1.15005117e-24,  1.15218653e-24, -6.60723735e-20]
+  >>> cent.array
+  array([-0.00100001, -0.003     ,  0.00150005])
 
-You can calculate the moment vector about the specified point for the
-specified locations as shown:
+**Example: Geometric centroid of the velocity inlets over multiple solvers**
 
 .. code-block:: python
 
-  >>> solver.fields.reduction.moment(
-  >>>   expression="['inlet1']",
-  >>>   locations=[solver.settings.setup.boundary_conditions.velocity_inlet["inlet2"]]
+  >>> cent = reduction.centroid(
+  >>>   locations=[VelocityInlets(settings_source=solver) for solver in [solver1, solver2]]
   >>> )
-  [ 1.15005117e-24,  1.15218653e-24, -6.60723735e-20]
+  >>> cent.array
```

Comment:
I'd prefer a simple Python list over a structure behind a private class.

The docstring of centroid says it returns a float.

Reply:
Done, Thank you.